windows test suite errors when linking

On link, I get the following error on tip:

```

link -nologo -incremental:no -debug -out:target/client.exe  common.obj client.obj target/release/rustls_ffi.lib advapi32.lib userenv.lib ws2_32.lib
rustls_ffi.lib(std-f87c887dcbebcf7e.std.06878a67-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_BCryptGenRandom referenced in function _ZN3std6thread5local4fast12Key$LT$T$GT$14try_initialize17h6f0eaa92973cfc73E
target\client.exe : fatal error LNK1120: 1 unresolved externals
mingw32-make: *** [Makefile.Windows:53: target/client.exe] Error 1120
```

See https://github.com/kevinburke/rustls-ffi/runs/4498117227?check_suite_focus=true for details.

--- a/Makefile.Windows
+++ b/Makefile.Windows
@@ -50,7 +50,7 @@ $(RUSTLS_LIB): src/lib.rs Cargo.toml
 	@echo
 
 target/%.exe: common.obj %.obj $(RUSTLS_LIB)
-	$(call link_EXE, $@, $^ advapi32.lib userenv.lib ws2_32.lib)
+	$(call link_EXE, $@, $^ advapi32.lib cfgmgr32.lib credui.lib gdi32.lib kernel32.lib msimg32.lib opengl32.lib secur32.lib user32.lib winspool.lib kernel32.lib ws2_32.lib bcrypt.lib advapi32.lib userenv.lib kernel32.lib msvcrt.lib)
 
 clean:
 	rm -f *.obj target/.rustc_info.json $(RUSTLS_LIB) vc1*.pdb

--- a/Makefile.Windows
+++ b/Makefile.Windows
@@ -42,7 +42,7 @@ test: all
 
 $(RUSTLS_LIB): src/lib.rs Cargo.toml
 	$(call green_msg, Building '$@')
-	cargo build --release
+	RUSTFLAGS="--print native-static-libs" cargo build --release
 	@echo
 
 %.obj: tests/%.c

